src/veh_interact.cpp

```diff
@@ -610,6 +610,23 @@ bool veh_interact::can_install_part() {
                           status_color( use_aid ), qual.obj().name.c_str(), lvl,
                           status_color( use_str ), str ) << "\n";
 
+    msg << _( "<color_white>Description</color>\n" );
+    std::string install_color = string_format( "<color_%1$s>",  status_color( ok || g->u.has_trait( trait_DEBUG_HS ) ) );
+    msg << install_color <<  sel_vpart_info->description << "</color>\n";
+
+    // borrowed from item.cpp and adjusted
+    const quality_id quality_jack( "JACK" );
+    const quality_id quality_lift( "LIFT" );
+    for (const auto& qual : sel_vpart_info->qualities) {
+        msg << install_color << "  Has level " << qual.second << " ";
```

Comment:
Would this `"Has level"` be translated?

Reply:
"has level" would need to be translated. I may have been too clever there.

I'll do some tests on the blank indent versus '>' issue.

Reply:
Is there an existing function that takes a long string and word wraps it? I don't want to write my own, because it would be bugg, but I couldn't find one on a quick search and I'm not familiar enough with C++ to know if there's a standard one.

Reply:
C++ style output stream, while very convenient for coder, is very troublesome for translator, since a long sentence get cut up into different pieces.
I would prefer printf() style with positional argument.

Reply:
Right, I realized that when Night-Pryanik asked about the translation. The new version will be closer to the item.cpp code: a single string_format for the "has level", a single string_format for the "and is rated at".

Reply:
Fixed with the third commit.